src/components/NcButton/NcButton.vue

```diff
@@ -554,7 +565,7 @@ export default {
 
 		// If we have a router-link, we wrap the button in it
 		if (this.to) {
-			return h(resolveComponent('router-link'), {
+			return h(RouterLink, {
```

Comment:
**issue (blocking):** `vue-router` is not a dependency or peerDependency of `@nextcloud/vue`

Reply:
Done